### DISCUSSION TITLE
i cant sign up!

### DISCUSSION THREAD

**Discussion by L1z1ng3r-sswe** [vercel/community#3152] [Help] [answered]
### Summary

i just wanted to sign up through github , but u (vercel) have  banned me for nothing! I cant deploy any of my repositories , even sign up! Pls unban me! 
This email (krasavachic@gmail.com)

### Steps to Reproduce

unban me  (krasavachic@gmail.com)

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the  email, phone number, and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
